05AB1E.py

```diff
@@ -2942,6 +2942,21 @@ def run_program(commands,
                     a = pop_stack(1)
                     stack.append(lcm(ast_int_eval(str(a)), ast_int_eval(str(b))))
 
+            elif current_command == ".\u00f8":
+                if stack:
+                    b = pop_stack(1)
+                    a = pop_stack(1)
+                else:
+                    a = pop_stack(1)
+                    b = pop_stack(1)
+                if type(a) is int:
+                    a = str(a)
+                if type(a) is str and type(b) is str:
+                    stack.append(b + a + b)
```

Comment:
if you'd prefer, I can remove the `type(b) is str` check, and use `str(b) + a + str(b)` here.